compiler/src/dmd/lexer.d

```diff
@@ -23,6 +23,7 @@ import core.stdc.time;
 
 import dmd.entity;
 import dmd.errors;
```

Comment:
Shouldn't the use of errorsink get rid of this import?

Reply:
Yes, it should. But there are a couple other cases where it is referenced directly that I will fix in a subsequent PR.